src/Core/Registration/GlobalOptimization.cpp

```diff
@@ -535,8 +537,25 @@ void GlobalOptimizationLevenbergMarquardt::
         timer_iter.Start();
         int lm_count = 0;
         do {
-            Eigen::MatrixXd H_LM = H + current_lambda * H_I;
-            Eigen::VectorXd delta = H_LM.ldlt().solve(b);
+			Eigen::MatrixXd H_LM = H + current_lambda * H_I;
```

Comment:
Can you change the indent from Tab to space?

Reply:
okay I will